### ISSUE TITLE
Consider adding an accessibility disclaimer to the Baseline badge

### ISSUE THREAD

**Issue by captainbrosset** [mdn/yari#10350]
### Summary

The context for this issue comes from https://github.com/web-platform-dx/web-features/issues/498.

Sorry, this is a long issue to read, but I think the gist of it can be understood by reading these comments:

1. https://github.com/web-platform-dx/web-features/issues/498#issue-2042572563
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858162816
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858511106
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858697605

In short, the [web-features repo](https://github.com/web-platform-dx/web-features) does not currently have a clear guideline around accessibility that's used to decide the Baseline status of a feature. We're discussing ways to change this, but it was suggested in the linked issue above that, in the meantime, it would be good for MDN to add a disclaimer to the Baseline badge.

See https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858697605 for an example of the proposal. Once expanded, the badge could have something like

```html
 There is no guarantee this feature is
 <a href="https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported">
  accessibility supported
 </a>, however, so support testing is recommended.
```

Because the UI rendering of the Baseline data is owned by MDN, I'm opening this issue for the MDN team to discuss and consider if they want to change the UI and add a note about accessibility.

### URL

N/A

### Reproduction steps

N/A

### Expected behavior

Accessibility should either be included in the baseline status for the feature, or a disclaimer that it's not should be present in the badge.

### Actual behavior

N/A

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Android

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by LeoMcA** [comment]
I don't think this is a good idea, for a number of reasons:

First, I reckon it'll fail to achieve what it sets out to do: people likely won't see this disclaimer, currently very few people actually open the Baseline banner: our metrics show between 2-3% of pageviews with a Baseline banner shown also have that Baseline banner toggled open.

Second, a11y support [isn't the only non-goal of Baseline](https://github.com/web-platform-dx/web-features/pull/519), do we add disclaimers for all those others in the banner as well? This will get rather overwhelming rather quickly, and fail on the aim of Baseline to be easily digestible.

Indeed, Baseline doesn't override general web development best practices: you should still test your code, regardless of what the compatibility data suggests. Perhaps some kind of "how to use Baseline" document would be a better place for a disclaimer of this type, along with the other non-goals.

**Comment by foolip** [comment]
Accessibility isn't a non-goal of Baseline, the delineation that was made in https://github.com/web-platform-dx/web-features/pull/519 is that if the browser does the wrong thing we consider that a user-affecting bug like any other, and it might lead to the feature not being considered ready. It's bugs outside of the browser, in the ATs, that we're not tracking.

**Comment by ddbeck** [comment]
> Indeed, Baseline doesn't override general web development best practices: you should still test your code, regardless of what the compatibility data suggests. Perhaps some kind of "how to use Baseline" document would be a better place for a disclaimer of this type, along with the other non-goals.

I would like to echo this sentiment. Perhaps [the Baseline glossary page](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility) is the right place to cover some of this stuff. That `Extra considerations` section could use some attention, along the lines of: don't forget to do accessibility, usability, performance, and other testing, or you'll be sorry whether or not there was a blue or green mark.

**Comment by caugner** [comment]
> I would like to echo this sentiment. Perhaps [the Baseline glossary page](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility) is the right place to cover some of this stuff. That _Extra considerations_ section could use some attention, along the lines of: don't forget to do accessibility, usability, performance, and other testing, or you'll be sorry whether or not there was a blue or green mark.

We discussed this internally and support Daniel's proposal to reflect this on the Baseline glossary page. 👍

**Comment by ddbeck** [comment]
I opened a PR for this in content: https://github.com/mdn/content/pull/33515
